src/components/ContentMapping/ContentMapping.ts

```diff
@@ -1,41 +1,48 @@
-import React from 'react';
```

Comment:
I dont know if your editor automatically formatted these but you should use single quotations for imports because it was originally that way and vscode should auto import with single quotations